packages/db/src/collection/mutations.ts

```diff
@@ -230,7 +230,8 @@ export class CollectionMutationsManager<
 
       // Apply mutations to the new transaction
       directOpTransaction.applyMutations(mutations)
-      directOpTransaction.commit()
+      // Errors still reject tx.isPersisted.promise; this catch only prevents global unhandled rejections
+      directOpTransaction.commit().catch(() => undefined)
 
       // Add the transaction to the collection's transactions store
       state.transactions.set(directOpTransaction.id, directOpTransaction)
```

Comment:
If the commit fails, do we still want to store this TX ID in the `state.transactions`?